From foo@baz Tue Apr  9 12:12:43 2002
To: Greg KH <greg@kroah.com>
Date: 02 Mar 2005 16:13:14 -08:00
From: khali@linux-fr.org
Subject: [PATCH] I2C: Change of i2c co-maintainer

Since I am working more actively than Philip (or anyone else, for that
matter) on the i2c subsystem these days, it would probably make sense
that I am listed as the co-maintainer instead of him.

Signed-off-by: Jean Delvare <khali@linux-fr.org>
Acked-by: Philip Edelbrock <phil@netroedge.com>
Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>


diff -Nru a/MAINTAINERS b/MAINTAINERS
--- a/MAINTAINERS	2005-03-02 16:31:35 -08:00
+++ b/MAINTAINERS	2005-03-02 16:31:35 -08:00
@@ -991,8 +991,8 @@
 I2C AND SENSORS DRIVERS
 P:	Greg Kroah-Hartman
 M:	greg@kroah.com
-P:	Philip Edelbrock
-M:	phil@netroedge.com
+P:	Jean Delvare
+M:	khali@linux-fr.org
 L:	sensors@stimpy.netroedge.com
 W:	http://www.lm-sensors.nu/
 S:	Maintained
